etc/conf/options.json

```diff
@@ -0,0 +1,5 @@
+{
+    "broker_uri": "amqp://10.10.2.1:30000",
+    "model_file": "./src/is_face_detector/haarcascade_frontalface_default.xml",
```

Comment:
add a message on options proto to configure all parameters related to face-detector